src/createHistoryEnhancer.js

```diff
@@ -0,0 +1,27 @@
+import { applyMiddleware } from 'redux';
+
+import createHistoryMiddleware from './createHistoryMiddleware';
+import createTransitionHookMiddleware from './createTransitionHookMiddleware';
+import ensureLocationMiddleware from './ensureLocationMiddleware';
+
+export default function createHistoryEnhancer(protocol, ...middlewares) {
+  return function historyEnhancer(createStore) {
+    return (...args) => {
+      const transitionHookMiddleware = createTransitionHookMiddleware();
+
+      const middlewareEnhancer = applyMiddleware(
```

Comment:
are you asking users to pass all middlewares through this our just the history specific ones?